lm_eval/models/huggingface.py

```diff
@@ -340,9 +334,18 @@ def _model_generate(
         max_tokens: int,
         stop: Optional[List[str]] = None,
     ) -> TokenSequence:
-        stopping_criteria = stop_sequences_criteria(self.tokenizer, stop)
+        input_ids = inputs["input_ids"][:, self.max_gen_toks - self.max_length :]
+        input_ids = input_ids.to(self.device)
+        
+        attention_mask = inputs["attention_mask"][
+            :, self.max_gen_toks - self.max_length :
+        ]
+        attention_mask = attention_mask.to(self.device)
+        
```

Comment:
`stopping_criteria`, used in the `self.model.generate` call, needs to be defined here. You may have accidentally deleted it.

Reply:
thanks, I accidentally deleted it while fixing the formatting.